.github/workflows/pr.yml

```diff
@@ -103,4 +113,4 @@ jobs:
 #    - name: Fail action when files change
 #      if: steps.changed_files.outputs.files_changed == 'true'
 #      run: |
-#        exit 1
+#        exit 1
```

Comment:
it seems no change here, please revert this.

Reply:
done